models/group-message.js

```diff
@@ -1,7 +1,39 @@
+const mongoose = require('mongoose');
 
+//Database connection
+mongoose.connect('mongodb://localhost/bears', { useNewUrlParser: true })
+  .then(() => console.log('Connected to MongoDB...'))
+  .catch(err => console.error('Could not connect to MongoDB...', err));
 
+//Message Object Schema (a sub document of the group message schema)
+const messageObjectSchema = mongoose.Schema({
```

Comment:
please use the 'new' keyword when using the mongoose.Schema 
```suggestion
const messageObjectSchema = new mongoose.Schema({
```